tpp-app/src/onboarding/PeriodLength.js

```diff
@@ -1,28 +1,44 @@
-import React from 'react';
-import { StyleSheet, ImageBackground } from 'react-native';
+import React, { useState } from 'react';
+import { StyleSheet, ImageBackground, TextInput } from 'react-native';
 import OnboardingBackground from '../../ios/tppapp/Images.xcassets/SplashScreenBackground.imageset/background.png'
 import { STACK_SCREENS } from './Confirmation';
 import { BackButton } from '../home/components/BackButtonComponent';
 import { NextButton, SkipButton } from './components/ButtonComponents';
 import { BodyText, TitleText } from './components/TextComponents';
-import { TwoButtonContainer, BackButtonContainer } from './components/ContainerComponents';
+import { TwoButtonContainer, BackButtonContainer, InputContainer } from './components/ContainerComponents';
+import { PostInitialPeriodLength } from '../services/OnboardingService';
 
 export default function PeriodLength ({ navigation }) {
+  const [periodLength, setPeriodLength] = useState(0)
   return (
     <ImageBackground  source={OnboardingBackground} style={styles.container}>
       <BackButtonContainer>
         <BackButton title="" onPress={() => {navigation.navigate(STACK_SCREENS["Get Started"])}}/>
       </BackButtonContainer>
+
       <TitleText>
         How long does your {'\n'} period usually last?
       </TitleText>
       <BodyText>
         This will help us make our {'\n'} reminders more accurate
       </BodyText>
 
+      <InputContainer>
+        <TextInput style={styles.input} 
+        placeholder="Tap to input" 
+        keyboardType="number-pad" 
```

Comment:
This can be fixed as part of your other ticket since it's not in the requirements for this one but I did not see a number pad keyboard when clicking on the input
<img width="292" alt="Screen Shot 2022-02-26 at 6 27 43 PM" src="https://user-images.githubusercontent.com/35851484/155862154-92055726-8af2-4a17-88b2-ebf5065a2e09.png">

Reply:
ooo yep for me the number pad would only pop up after i enable the keyboard with cmd k i will look more into this